util/dockerutil/client.go

```diff
@@ -63,6 +63,23 @@ func (c *Client) LoadImage(ctx context.Context, name string, status progress.Wri
 	}, nil
 }
 
+func (c *Client) Features(ctx context.Context, name string) map[Feature]bool {
+	features := make(map[Feature]bool)
+	if dapi, err := c.API(name); err == nil {
```

Comment:
In a follow-up this function could do caching these return values for `bake` and if we add any more `Feature` values we probably would need to ask for a specific one to make sure expensive call like `info` is not done needlessly.